src/components/LocationForm.js

```diff
@@ -25,7 +26,7 @@ function LocationForm() {
 
     if (validate()) {
       console.log("Submitting form with:", { title, description, address, hashtags, category, imageLink });
-      let res = await fetch("http://localhost:3000/api/locations", {
+      let res = authenticatedFetch("http://localhost:3000/api/locations", {
```

Comment:
You accidentally deleted the `await` before `authenticatedFetch` . Other than that it looks very good!